provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -291,4 +291,6 @@ def _create_node(self, machine_id: str) -> NodeDesc:
             # TODO in the future the value must be taken from a correct
             # ConfStore key (it doesn't exist now).
             s3_instances=int(
-                store.get(f'server_node>{machine_id}>s3_instances')))
+                store.get(f'server_node>{machine_id}>s3_instances')),
+            m0_clients=int(
+                store.get(f'server_node>{machine_id}>m0_clients')))
```

Comment:
@supriyachavan4398 Does this key exist in ConfStore nowadays?

Reply:
No, Manually I added this for testing.

Reply:
Yes, I saw your code. I meant a bit different thing.

The list of keys that ConfStore can potentially have is confirmed by other people. As far as I understand the latest "snapshot" of all confirmed keys can be seen here https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys
And I don't see `m0_clients` there. Which means that we'll need to initiate the corresponding discussions...

Reply:
Okay Sure, I'll look into it.

Reply:
@supriyachavan4398 We can modify our code such that if key is not present in confstore then we will use default hardcoded value.

Reply:
@supriyachavan4398 both [EOS-21188](https://jts.seagate.com/browse/EOS-21188) and [the doc](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys) mention `cortx>software>motr>service>client_instances`.
Why do you refer to `m0_clients`?

Reply:
I addressed this comment and m0_clients replaced with client_instances.